packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -121,6 +124,7 @@ export class DefaultCrudRepository<
     );
 
     this.modelClass = this.definePersistedModel(entityClass);
+    this.inclusionResolvers = new Map<string, InclusionResolver<T, Entity>>();
```

Comment:
Remove it if https://github.com/strongloop/loopback-next/pull/3517/files#r313934062 is resolved.